HOW_TO.md

```diff
@@ -84,39 +84,39 @@ The packages listed will be installed using `apt-get` as part of the Docker buil
 ### Test your algorithm <a name="test"/>
 A testing script named `testing.py` is provided for testing your algorithm.
 It checks whether the configuration is correct for testing the files by making sure that the arguments in algorithm_rgb as well as the image files are in the correct format
-What isn't provided in the template repository are the plot-level RGB images to test against.
-It's expected that you will either provide the images or use a standard set that can be downloaded from [Google Drive](https://drive.google.com/file/d/1xWRU0YgK3Y9aUy5TdRxj14gmjLlozGxo/view?usp=sharing).
 
 The testing script requires `numpy` and `gdal` to be installed on the testing system.
 
-For example, if your files reside in `/user/myself/test_images` the command to test could be the following:
+If your files reside in `/user/myself/test_images` the command to test could be the following:
 ```./testing.py /user/myself/test_images```
 
+What isn't provided in the template repository are the plot-level RGB images to test against.
+It's expected that you will either provide the images or use a standard set that can be downloaded from [Google Drive](https://drive.google.com/file/d/1xWRU0YgK3Y9aUy5TdRxj14gmjLlozGxo/view?usp=sharing).
+
 The testing script expects to have either a list of source plot image files, or a folder name, or both specified on the command line.
```

Comment:
What happened to running testing.py from within a docker container? I thought that it was useful, otherwise I would have suggested that it be removed.